lib/flutter_neat_and_clean_calendar.dart

```diff
@@ -370,7 +370,7 @@ class _CalendarState extends State<Calendar> {
             children: <Widget>[
               todayIcon ?? Container(),
               Text(
-                displayMonth,
+                displayMonth.toUpperCase(),
```

Comment:
Please remove the `.toUpperCase` there. This changes the behavior of the library. I don't think, everybody likes to show up the months in uppercase letters.